numpy/lib/_function_base_impl.py

```diff
@@ -1713,6 +1713,21 @@ def unwrap(p, discont=None, axis=-1, *, period=2*pi):
     difference from their predecessor of more than ``max(discont, period/2)``
     to their `period`-complementary values.
 
+    Output `q` satisfies:
+
+     1. ``q[0] == p[0]``.
+     2. :math:`\forall i`. ``(p[i] - q[i]) % period == 0``.
+     3. :math:`\forall i>0`. ``abs(p[i] - p[i-1]) < discont`` -> 
+      ``q[i] - q[i-1] == p[i] - p[i-1]``.
+     4. :math:`\forall i>0`. ``abs(p[i] - p[i-1]) >= discont`` ->
+       a. ``abs(q[i] - q[i-1]) <= abs(period/2)``.
+       b. ``(p[i] - p[i-1]) % period == (period/2)`` ->
+        1. ``period > 0`` -> ``q[i] > q[i-1]`` iff ``p[i] > p[i-1]``.
+        2. ``period < 0`` -< ``q[i] < q[i-1]`` iff ``p[i] > p[i-1]``.
```

Comment:
Make sure you align the second and third levels with the start of the previous text, like:
```
1. Hello
   This is the continuation
   * with a bullet point
     that is also continued.
```
I am not sure if sphinx will like 3 levels of numbering like this (or even like alphabetic numbering).

You can try if `spin docs` works for you locally, to have a look (should work, although you may have to install the dependencies).

Also there is an "ci/circleci: build artifact" CI job, if you click it, you will see the rendered documentation from the PR.

---

Have to read it closer, but I think we should move this into/modifying the "Notes" section at the bottom.

Some random notes (will have to iterate once more, maybe with someone else probably):
* "iff" is not generally well known, so we should avoid it, many will read it as a typo.
* I would probably leave out the note about the axis.  This is a common pattern in NumPy API, IMO.  Also about the comparisons, all operations in NumPy are limited by Numerical precision.  (Unless it can fail worse here than for, say a `sum()` calculation.)